exercises/saddle-points/.meta/template.j2

```diff
@@ -0,0 +1,58 @@
+"""Tests for the saddle-points exercise
+
+Implementation note:
+The saddle_points function must validate the input matrix and raise a
+ValueError with a meaningful error message if the matrix turns out to be
+irregular.
+"""
+{%- import "generator_macros.j2" as macros with context -%}
+
+{% macro test_case(case) -%}
+    {%- set input = case["input"] -%}
+    {%- set expected = case["expected"] -%}
+    def test_{{ case["description"] | to_snake }}(self):
+        {% if input["matrix"][0] -%}
+        matrix = {{ input["matrix"] }}
+        {%- else -%} matrix = [] {% endif %}
```

Comment:
For readability, please adjust the spacing on this:
```suggestion
        {%- else -%}
        matrix = []
        {% endif -%}
```

Reply:
Cleaned up.